src/css/custom.scss

```diff
@@ -318,3 +318,10 @@ div[class^='codeBlockTitle'] {
 [class^='docItemContainer'] {
   margin: 0 8px;
 }
+
+/* mobile navigation is always in dark mode,
+   so keep active state consistent, regardles
+   of site color mode state */
+.navbar-sidebar .menu__link--active {
```

Comment:
could we move this to navbar.scss?

Reply:
Sure. I almost did, but paused as it's the `navbar-sidebar` class and we have both navbar & sidebar `.scss` files 😆

Reply:
![image](https://user-images.githubusercontent.com/168135/212421579-9b603e18-5ee0-40a8-be4a-432659567dde.png)